inc/devicesimcard.class.php

```diff
@@ -0,0 +1,204 @@
+<?php
+/*
+ * @version $Id$
+ -------------------------------------------------------------------------
+ GLPI - Gestionnaire Libre de Parc Informatique
+ Copyright (C) 2015 Teclib'.
+
+ http://glpi-project.org
+
+ based on GLPI - Gestionnaire Libre de Parc Informatique
+ Copyright (C) 2003-2014 by the INDEPNET Development Team.
+
+ -------------------------------------------------------------------------
+
+ LICENSE
+
+ This file is part of GLPI.
+
+ GLPI is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+
+ GLPI is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ --------------------------------------------------------------------------
+ */
+
+/** @file
+* @brief
+*/
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access directly to this file");
+}
+
+/// Class DeviceProcessor
+class DeviceSimcard extends CommonDevice {
+
+   static protected $forward_entity_to = array('Item_DeviceSimcard', 'Infocom');
+
+   static function getTypeName($nb=0) {
+      return _n('Simcard', 'Simcards', $nb);
+   }
+
+   /*
+   function getAdditionalFields() {
+
+      return array_merge(parent::getAdditionalFields(),
+                         array(array('name'  => 'frequency_default',
+                                     'label' => __('Frequency by default'),
+                                     'type'  => 'text',
+                                     'unit'  => __('MHz')),
+                               array('name'  => 'frequence',
+                                     'label' => __('Frequency'),
+                                     'type'  => 'text',
+                                     'unit'  => __('MHz')),
+                               array('name'  => 'nbcores_default',
+                                     'label' => __('Number of cores'),
+                                     'type'  => 'integer'),
+                               array('name'  => 'nbthreads_default',
+                                     'label' => __('Number of threads'),
+                                     'type'  => 'integer')
+                           ));
+   }
+   */
+
+   function getSearchOptionsNew() {
+
+      $tab                 = parent::getSearchOptionsNew();
+
+      $tab[] = [
```

Comment:
i don't get the point of theses searchoptions for a simcard.
frequency, nb cores ?